<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PNRF ATP Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_LoadFile">
           <property name="text">
            <string>Load PNRF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Connect_ATP">
           <property name="text">
            <string>Connect ATP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_ATP_Run">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupnrf_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_file"/>
    <addaction name="actionClear_file"/>
   </widget>
   <widget class="QMenu" name="menuATP">
    <property name="title">
     <string>ATP</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="actionRebuild_ATP"/>
   </widget>
   <addaction name="menupnrf_File"/>
   <addaction name="menuATP"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load PNRF File</string>
   </property>
  </action>
  <action name="actionClear_file">
   <property name="text">
    <string>Clear PNRF File</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionSettings_2">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionShow_Graphics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Graphics</string>
   </property>
  </action>
  <action name="actionRebuild_ATP">
   <property name="text">
    <string>Rebuild ATP</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
